runtime/doc/lua.txt

```diff
@@ -2372,4 +2372,29 @@ read({path})                                               *vim.secure.read()*
         (string|nil) The contents of the given file if it exists and is
         trusted, or nil otherwise.
 
+    See also: ~
+        vim.secure.trust()
+        :trust
```

Comment:
Looks like the docgen script doesn't automatically create links here, so we'll have to do that in the docstring.

Reply:
Done in 4b1d99b1cdd0ce92aa3867d0568d1c6e9a61e8df